packages/edit-site/src/components/block-editor/index.js

```diff
@@ -173,6 +195,13 @@ export default function BlockEditor() {
 			useSubRegistry={ false }
 		>
 			{ hasPageContentLock && <PageContentLock /> }
+
+			<SetRootBlockEditingMode
+				mode={
+					templateType === `wp_navigation` ? 'contentOnly' : 'default'
+				}
+			/>
```

Comment:
@noisysocks This line no longer seems to work following a rebase. I checked that the type is correctly set to `wp_navigation` but inside the Nav _block_ itself `useBlockEditingMode` continues to resolve to `default` instead of `contentOnly`.